test/scripts/jenkins_win_selenium.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/bash
+
+source "$(dirname $0)/../../src/dev/ci_setup/setup.sh"
+ 
+# Install Dir
+installDir=$(pwd)
+mkdir -p "$installDir" 
+
+# Get Version
+pkg_version=$(cat package.json | grep "\"version\"" | cut -d ':' -f 2 | tr -d ",\"\ +")
+version=${TEST_ES_BRANCH:=${pkg_version}-SNAPSHOT}
+
+# Download Kibana Package 
```

Comment:
I get that we don't support building on windows, so we don't have a lot of options here, but are these tests just going to run on a timer or are we going to run them as a part of the PR/master tests?